js/main.js

```diff
@@ -35,19 +41,13 @@ function getRandomArbitrary(num1, num2, fixed) {
   }
 }
 
-
-getRandomArbitrary(3, 5.1, 5);
 let numImg = 0;
 
 const createImgNumber = function () {
   numImg++;
   return numImg < 10 ? '0' + numImg : numImg;
```

Comment:
По правилам академии следует сделать интерполяцию, вместо сложения.
```suggestion
  return numImg < 10 ? `0${numImg}` : numImg;
```